SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -276,6 +276,13 @@ void USpatialSender::SendComponentUpdates(UObject* Object, FClassInfo& Info, USp
 		if (!NetDriver->StaticComponentView->HasAuthority(EntityId, Update.component_id))
 		{
 			UE_LOG(LogSpatialSender, Warning, TEXT("Trying to send component update but don't have authority! Update will not be sent. Component Id: %d, entity: %lld"), Update.component_id, EntityId);
+			UE_LOG(LogSpatialSender, Warning, TEXT("Queueing the update, to send upon receiving authority."));
```

Comment:
Let's either remove this line or reduce it to `Log`. Nothing is "wrong" so shouldn't be a warning, and some of our customers will want to run at 0 warnings.

If you keep it, add a bit more info like component id, entity etc.